test/src/core/testPageObjectApi.js

```diff
@@ -94,7 +94,6 @@ describe('test PageObjectApi', function () {
     assert.ok('click' in page);
     assert.ok('waitForElementPresent' in page);
     assert.ok('end' in page);
-    assert.ok('switchWindow' in page);
```

Comment:
I think this is actually a bug. I have renamed switchWindow to switchToWindow and kept an alias, but it looks like aliases are not loaded onto page objects.

Reply:
but also, using switchWindow in the context of a page doesn't make much sense to begin with, so we can drop it.